Journal entries respect whitespace in content (#392)

harokb opened the pull request:
This should fix issue #392. I tested the solution by changing the HTML directly in Chrome, and it worked, but I haven't tested through Homestead, but it should work since the problem is really simple.

djaiss commented:
Thanks, but your fix breaks the layout.

![image](https://user-images.githubusercontent.com/61099/27538397-bef6c04c-5a45-11e7-8235-ad5d1c0ae250.png)

harokb commented:
My bad, I added a second opening tag instead of a closing tag. It's now fixed. 

I tested it locally, and the layout seems correct:

![image](https://user-images.githubusercontent.com/4773657/27542839-5ea46060-5a80-11e7-92ee-88e65dcea9d3.png)

djaiss commented:
Thanks a lot!
